ocserv-authentication-ad-googleauth.md

```diff
@@ -0,0 +1,105 @@
+## How to setup ocserv for Microsoft AD Authentication and Google Authenticator as a 2FA
+
+Author: Dionis Pelivan
+
+
+### Scope
+
+This Recipe provides step by step instructions on how to install, configure,
+and test Microsoft AD Authentication for Openconnect Server. This recipe focuses on
+generic installation instructions, from packages available on Openconnect server.
+No precompiled binary packages will be used, therefore this recipe was tested 
```

Comment:
Not sure what this sentence means by "No precompiled binary packages will be used". Does it mean you didn't use the epel7 ocserv package?

Reply:
@dpelivan ?